.projenrc.ts

```diff
@@ -863,7 +863,7 @@ cli.gitignore.addPatterns('build-info.json');
 const cliPackageJson = `${cli.workspaceDirectory}/package.json`;
 
 cli.preCompileTask.prependExec('./generate.sh');
-cli.preCompileTask.prependExec('ts-node scripts/user-input-gen.ts');
+cli.preCompileTask.prependExec('ts-node --prefer-ts-exts scripts/user-input-gen.ts');
```

Comment:
This was need to make the build more manageable. Otherwise the script would always use the js file and you would need to manually delete it.